packages/web-twig/src/Resources/components/FileUploader/__tests__/__fixtures__/fileUploaderInput.twig

```diff
@@ -24,6 +24,20 @@
     multiple
 />
 
+<!-- Render with dynamic multiple attribute -->
+<FileUploaderInput
+    accept="*.avif,*.jpg,*.png,*.webp"
+    id="example-input-html-attributes"
+    multiple={false}
```

Comment:
Or
```suggestion
    multiple={ false }
```
?

I don't know what is your formatting preference across the Twig files. :)

Reply:
This was a quick try. So I do not bother with this at the beginning 😅

Reply:
It seems we don't use spaces inside single curly braces.

Reply:
Well, I'm not sure. 🙊

<img width="1096" alt="image" src="https://github.com/lmc-eu/spirit-design-system/assets/33354913/b086bccf-7fd8-4e80-82ce-d177db68dbd8">

Reply:
Fixed.